templates/frontEndLayout.html

```diff
@@ -255,13 +265,13 @@
             <div style = "text-align: left;">Alfalfa Sprouts</div>
         </td>
         <td>
-            <div class = "border">$1</div>
+            <div class = "border" onclick = "sell('a')">$1</div>
```

Comment:
Rather than put this "$1" in the html, you should have a `reset()` or `initialize()` function that sets all these values.  Right now if we changed the price in GAME_CONFIG, it would still show $1 but pay out at the new price.